content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -1,121 +1,83 @@
 # Sharing Experiments
 
-There are two types of remotes that can store experiments. Git remotes are
-distributed copies of the Git repository, for example on GitHub or GitLab.
+If your team uses Git server or hosting (e.g. GitHub, GitLab, etc.) to
+collaborate on projects, you can also use them to save and share DVC
+Experiments.
 
-[DVC remotes](/doc/command-reference/remote) on the other hand are
-storage-specific locations (e.g. Amazon S3 or Google Drive) which we can
-configure with `dvc remote`. DVC uses them to store and fetch large files that
-don't normally fit inside Git repos.
+> You will need both [Git remotes] and DVC
+> [remote storage](/doc/command-reference/remote) for this.
 
-DVC needs both kinds of remotes for backing up and sharing experiments.
+[git remotes]: https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
 
-Experiment files that are normally tracked in Git (like code versions) are
-shared using Git remotes, and files or directories tracked with DVC (like
-datasets) are shared using DVC remotes.
+<details>
 
-> See [Git remotes guide] and `dvc remote add` for information on setting them
-> up.
+## ⚙️ Expand to learn more.
 
-[git remotes guide]:
-  https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
+Sharing experiments is similar to [sharing regular project data] and artifacts
+by synchronizing from remotes. DVC-tracked data, models, etc. are in your
+project's <abbr>cache</abbr> and thus will be transferred to/from
+[remote storage](/doc/command-reference/remote) (e.g. Amazon S3 or Google
+Drive). Small files like experimental code and
+[DVC metafiles](/doc/user-guide/project-structure) files are uploaded or
+downloaded with Git automatically as needed.
```

Comment:
probably not automatically, `dvc exp pull` is needed, right?

again, ideally, we want to be explicit that it's not the regular way you would use Git - it's based on custom refs, it doesn't affect / pollute history, etc

Reply:
> probably not automatically... be explicit that it's not the regular way you would use Git

Changed "automatically as needed" for "by DVC" in this details section. The other details are explained further down.